### DISCUSSION TITLE
Bug in docker-compose.yml file

### DISCUSSION THREAD

**Discussion by TheVerga** [dasniko/keycloak-extensions-demo#17] [General]
Not sure if this is the right place to write, but I've found a bug in the docker-compose.yml file.
I wanted to open an issue and a pull-request to fix it, but it does not seem possible, so here I am :)

The volume binding
```
- ./conditional-headers-authenticator/target/dasniko.keycloak-conditional-headers-authenticator.jar:/opt/keycloak/providers/conditional-headers-authenticator.jar
```
is incorrect, because it refers to an unexisting folder (maybe it was refactored in the past?)
The correct binding should be:
```
- ./conditional-authenticators/target/dasniko.keycloak-conditional-authenticators.jar:/opt/keycloak/providers/conditional-authenticators.jar
```
Hope this helps

**Comment by dasniko** [comment]
Thanks 🙏 , yes, I refactored some resources in the past weeks.
As all this is just for demo purposes, I assume that the people who are using this, are also thinking a bit about what they're doing. Like you do! 😄
